user-service/src/main/java/rs/edu/raf/banka1/requests/customer/CreateCustomerRequest.java

```diff
@@ -12,4 +12,5 @@
 public class CreateCustomerRequest {
     private CustomerData customer;
     private AccountData account;
+    //private BankAccountRequest account;
```

Comment:
Ovo ne bi trebalo da bude ovde?

Reply:
e to je okej slucajno mi ostalo zakomentarisano jer u AccountData imaju sva polja za BankAcc
obrisacu taj comment

Reply:
Oukee.